rome.json

```diff
@@ -4,7 +4,7 @@
     "enabled": false
   },
   "files": {
-    "ignore": ["built/", "**/browser_polyfill.js"]
```

Comment:
Since we're targeting [Manifest v3 in Chrome, it supports Promises as well](https://developer.chrome.com/docs/extensions/mv3/intro/), and we were just using it for the `browser.*` support, which I handled differently via `globalThis`.